server/routes/api/api-user-routes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+/**
+ * user routes
+ */
+module.exports = function(controllers, middlewares) {
```

Comment:
__[eslint]__
* Error - 'module' is not defined. (no-undef)
* Error - 'middlewares' is defined but never used (no-unused-vars)